frontend/packages/dev-console/integration-tests/tests/pipeline.scenario.ts

```diff
@@ -120,4 +133,70 @@ describe('Pipeline', async () => {
       true,
     );
   });
+  it('start pipeline scenario', async () => {
```

Comment:
Please add a description of the test that this function performs. Also - please add comments to the code as it is not always possible to understand the action that the "user" is performing in the UI .